pkg/block/device.go

```diff
@@ -35,14 +35,15 @@ type Device interface {
 
 // Partition struct represents a partition with its commonly configurable values, size in MiB
 type Partition struct {
-	Name            string
-	FilesystemLabel string
-	Size            uint
-	FS              string
-	Flags           []string
-	MountPoints     []string
-	Path            string
-	Disk            string
+	Name        string
+	Label       string
```

Comment:
Renamed FilesystemLabel to Label as this is the common notation in linux in fstab or kernel args (e.g. `LABEL=SYSTEM`) to identify the target device.